packages/shared/lib/clients/orchestrator.ts

```diff
@@ -728,8 +728,26 @@ export class Orchestrator {
 
         // Orchestrator
         const scheduleName = ScheduleName.get({ environmentId, syncId });
-        const frequencyMs = ms(interval as StringValue);
-        const res = await this.client.updateSyncFrequency({ scheduleName, frequencyMs });
+
+        const cleanInterval = this.cleanInterval(interval);
+        if (isOrchestrator && cleanInterval.isErr()) {
+            errorManager.report(cleanInterval.error, {
+                source: ErrorSourceEnum.CUSTOMER,
+                operation: LogActionEnum.SYNC_CLIENT,
+                environmentId,
+                metadata: {
+                    syncName,
+                    scheduleName,
+                    interval
+                }
+            });
+            return Err(cleanInterval.error);
+        }
+        let res: Result<void> = Ok(undefined);
+        if (cleanInterval.isOk()) {
```

Comment:
It is a bit convoluted but this `if` is only necessary because of the flag. Will go away with temporal

Reply:
I don't follow completely but since it blocking for a customer -- approving.

Reply:
Looked at this again. cleanInterval is a weird name to me. At first I read it as clearInterval. But the function is checking the interview and making sure it is valid right? 

validateInterval?

Reply:
formatInterval

Reply:
My initial though was it is a function that takes Nango interval and clean it for `ms` but I am happy to rename it. I will do that when I remove the flag

Reply:
Got it. Thanks